pkg/cmd/server/start/start_master.go

```diff
@@ -547,7 +547,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 		kc.RunScheduler()
 		kc.RunReplicationController(rcClient)
 		kc.RunJobController(jobClient)
-		kc.RunHPAController(hpaKClient)
+		kc.RunHPAController(hpaOClient, hpaKClient, oc.Options.PolicyConfig.OpenShiftInfrastructureNamespace)
```

Comment:
@liggitt Like this (and I think this is a good default) or should we have a separate knob?  I'd propose that as a followup if we want one.